benchmark/benchmarks/transfer/transfer.go

```diff
@@ -398,6 +402,8 @@ func Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&flagFaucetURL, cfgFaucetURL, "", "Faucet private endpoint URL")
 	cmd.Flags().BoolVar(&flagWatchNewHead, cfgWatchNewHead, false, "Subscribe for `newHeads` events")
 	cmd.Flags().BoolVar(&flagQueryNonces, cfgQueryNonces, false, "Query explicitly for account nonces")
+	cmd.Flags().StringVar(&flagFundingAccount, cfgFundingAccount, "533d62aea9bbcb821dfdda14966bb01bfbbb53b7e9f5f0d69b8326e052e3450c", "Hex encoded funding account key")
+	cmd.Flags().StringVar(&flagFundingAccountAddress, cfgFundingAccountAddress, "0x7110316b618d20d0c44728ac2a3d683536ea682b", "Funding account address")
 
 	for _, v := range []string{
 		cfgFaucetURL,
```

Comment:
you sure you don't need to add `cfgFundingAccount` and `cfgFundingAccountAddress`?

Reply:
thanks for noticing :-)